src/chunk.c

```diff
@@ -2305,7 +2304,7 @@ ts_chunk_data_nodes_copy(const Chunk *chunk)
 }
 
 Chunk *
-ts_chunk_copy(const Chunk *chunk)
+ts_chunk_duplicate(const Chunk *chunk)
```

Comment:
Why this change? Is it to avoid confusion with the DDL function with similar name? 

I worry that "duplicate" is somewhat confusing. Think we also use "copy" semantics for other internal objects.

Reply:
I wanted to reuse chunk_copy name and found this collision which I found a bit confusing regarding its name and what it does. Thought that naming it as a duplicate is more representative actually

Reply:
+1 for keeping the earlier name. `duplicate` is confusing.